src/utils/themeInjector.js

```diff
@@ -34,6 +34,26 @@ module.exports = class Injector {
             }
         }
     }
+
+    injectZoom(zoom) {
```

Comment:
I dont see a reason to put this in theme injector, id just create a new function in main.js

Reply:
Yeah, that was the part I least liked about my choices. I liked it better now on main.js.